ShowBuildMenu.sh

```diff
@@ -190,28 +190,7 @@ getAsyncGeneratorPath(){
 	fi
 
 	cd Tools
-
-	if [ ! -f nuget.exe ]
-	then
-		wget https://dist.nuget.org/win-x86-commandline/latest/nuget.exe
-	fi
-
-	async_generator_path="CSharpAsyncGenerator.CommandLine.$(cat packages.config | grep id=\"CSharpAsyncGenerator.CommandLine | cut -d\" -f4)/tools"
-
-	if [ ! -d $async_generator_path ]
-	then
-		mono nuget.exe install
```

Comment:
Still need `nuget install` part, I believe.

Reply:
You are right, I reverted it. Unfortunately .NET Core does not support packages.config, so mono is still required here.

Reply:
Maybe should we work around this by replacing this package.config by a "package.csproj"?

Reply:
I've added `packages.csproj` and used it only for the generator as I don't quite understand where other tools like `vswhere` are used.